app/controllers/books_controller.rb

```diff
@@ -0,0 +1,74 @@
+# Manages the "Book" model
+class BooksController < ApplicationController
```

Comment:
Might want to make this a subclass of AdminController. Then whatever auth logic we decide to add, will apply to all parts of the admin UI.